oaiharvest/harvest.py

```diff
@@ -26,15 +26,23 @@
   -u YYYY-MM-DD, --until YYYY-MM-DD
                         harvest only records added/modified up to this date.
   -s SET, --set SET     harvest only records within this set
+  -b HH:MM HH:MM, --between HH:MM HH:MM
```

Comment:
Does this actually work with shell command line parsing? Argument values aren't usually allowed to contain whitespace.

Reply:
Yes, the `nargs=2` argument takes care of this.

https://github.com/bloomonkey/oai-harvest/blob/051668117f33822f4d6db5afe28b1a1325320afe/oaiharvest/harvest.py#L423-L427